frontend/src/Components/SignUp.js

```diff
@@ -0,0 +1,66 @@
+import ReactDOM from "react-dom";
+import styles from "./SignUp.module.css";
+import { useState } from "react";
+import { FaTimes as Cross } from "react-icons/fa";
+
+const modalRoot = document.querySelector("#modal-root");
+
+const SignUp = ({ cancel }) => {
+  let [user, setUser] = useState(undefined);
+  let [password, setPassword] = useState(undefined);
+  let [passwordC, setPasswordC] = useState(undefined);
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    console.log("Creating account");
+    // // check if passwords are the same
```

Comment:
Maybe we could add some username checks

Probably just regex strings

Reply:
Yeah we can do that when we have an endpoint to go to.